app/model/sdl/Abstract/Model.js

```diff
@@ -1033,6 +1032,54 @@ SDL.SDLModel = Em.Object.extend({
    */
   imageCheckList: {},
 
+  /**
+   * @description Checks images in request provided inside 'objectsWithImages' param.
+   * @param requestID {Integer} Id of request
+   * @param callback {Function} User defined callback 
+   * @param objectsWithImages {Array} Array of data structures, specific to request,
+   *                                  that contain images.
+   * @return {Boolean} Returns true, if all images have valid extension. 
+   *                   Returns false, if at least 1 image has invalid extension.
+   */
+  checkImagesInRequest: function(requestID, callback, objectsWithImages) {
```

Comment:
@YarikMamykin 
```suggestion
  validateImagesExtensions: function(requestID, callback, arrayToCheck) {
```

Reply:
@AKalinich-Luxoft current method not only validates images by extensions but also starts async validation of images by path. Maybe it is more suitable to change method name to `validateImagesInRequest` ?

Reply:
@YarikMamykin yep, do it

Reply:
@AKalinich-Luxoft done in https://github.com/LuxoftSDL/sdl_hmi/pull/9/commits/f971050f986169148ea00d0f80171ba3f3afbe8b https://github.com/LuxoftSDL/sdl_hmi/pull/9/commits/d81dfea73b2458f24b3f7fdbb73b9b96897ee506 https://github.com/LuxoftSDL/sdl_hmi/pull/9/commits/17eef4ffc6314311afb5bb1ea68f4fe7efef922c https://github.com/LuxoftSDL/sdl_hmi/pull/9/commits/7cc697e239cf0fbca01509a0d4078658f35170b7